lib/validation.ts

```
@@ -0,0 +1,318 @@
+    );
+  }
+}
+
+/**
+ * Sanitizes a string by removing potentially dangerous characters
+ * Use this for user-generated content that will be displayed
+ */
+export function sanitizeString(input: string): string {
+  // Remove control characters and null bytes
+  return input
+    .replace(/[\x00-\x1F\x7F]/g, '') // Remove control characters
```

Biome (2.1.2)

[error] Unexpected control character in a regular expression.

Control characters are unusual and potentially incorrect inputs, so they are disallowed.

(lint/suspicious/noControlCharactersInRegex)